apps/src/templates/teacherHomepage/Courses.jsx

```diff
@@ -80,6 +86,7 @@ const Courses = React.createClass({
               {i18n.courseExplorerDescription()}
             </div>
             <ProtectedStatefulDiv ref="courseExplorer"/>
+            <div style={styles.spacer}>.</div>
```

Comment:
I'd normally use a `&nbsp;` for space before an invisible `.` if I understand this correctly.